pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -155,7 +155,7 @@ func NewManager(cpuPolicyName string, reconcilePeriod time.Duration, machineInfo
 		if err != nil {
 			return nil, err
 		}
-		klog.Infof("[cpumanager] detected CPU topology: %v", topo)
+		klog.InfoS("[cpumanager] detected CPU topology", "topology", topo)
```

Comment:
```suggestion
		klog.InfoS("detected CPU topology", "topology", topo)
```

Please remove all log line prefixes. (I won't comment on every line, only the ones that have other comments.)